addons/account/models/account_move_line.py

```diff
@@ -3289,6 +3337,41 @@ def _filter_aml_lot_valuation(self):
         self.ensure_one()
         return self.move_id.state == 'posted'
 
+    def get_lines_grouped_by_section(self):
+        """
+        Return a tax-wise summary of sale order lines linked to section.
+        Groups lines by their tax IDs and computes subtotal and total for each group.
+        """
+        self.ensure_one()
+
+        section_lines = self.section_line_child_ids + self.move_id.invoice_line_ids.filtered(lambda line: line.display_type == 'line_subsection' and line.section_line_parent_id == self).section_line_child_ids
```

Comment:
depending on the decision about storing/one2many - we could use something ugly like  `gp + gp.child_ids + gp.child_ids.child_ids`

self.search([('id', 'child_of', self)])